manifests/params.pp

```diff
@@ -37,7 +37,11 @@
   $db_username = 'foreman'
   # Generate and cache the password on the master once
   # In multi-puppetmaster setups, the user should specify their own
+
+  # TODO: Use Sensitive, as soon, as the Installer supports it
+  #$db_password = Sensitive(extlib::cache_data('foreman_cache_data', 'db_password', extlib::random_password(32)))
```

Comment:
I'd prefer to avoid TODOs in the code for this and rather open an issue in the repo.

Reply:
I removed the TODOs and amended the Commit.